src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -561,22 +592,44 @@ START_SECTION(PeakShapeMetrics calculatePeakShapeMetrics(
   PeakIntegrator::PeakShapeMetrics psm;
   pa = ptr->integratePeak(chromatogram, left, right);
   psm = ptr->calculatePeakShapeMetrics(chromatogram, left, right, pa.height, pa.apex_pos);
-  TEST_REAL_SIMILAR(psm.width_at_5, 0.231263425125414)
-  TEST_REAL_SIMILAR(psm.width_at_10, 0.134762234301732)
-  TEST_REAL_SIMILAR(psm.width_at_50, 0.0595791540757924)
-  TEST_REAL_SIMILAR(psm.start_position_at_5, 2.51268515480125)
-  TEST_REAL_SIMILAR(psm.start_position_at_10, 2.63222565817823)
-  TEST_REAL_SIMILAR(psm.start_position_at_50, 2.65391757114759)
-  TEST_REAL_SIMILAR(psm.end_position_at_5, 2.74394857992666)
-  TEST_REAL_SIMILAR(psm.end_position_at_10, 2.76698789247996)
-  TEST_REAL_SIMILAR(psm.end_position_at_50, 2.71349672522338)
+  TEST_REAL_SIMILAR(psm.width_at_5, 0.15955)
```

Comment:
why are the values now so different? is this expected?

Reply:
Yes, because we are now taking the point's position __before__ the percentage's goal. Not the one closest.
The one to the left in case of the left half, and the one to the right in case of the right half.